mmdt/templates/post_detail.html

```diff
@@ -1,47 +1,61 @@
-{% extends 'base.html' %}
-{% load crispy_forms_tags %}
-{% block content %}
+{% extends 'base.html' %} {% load crispy_forms_tags %} {% block content %}
 <section>
-<div class="card mb-3">
-  <img class="card-img" src="{{ post.image.url }}" alt="Card image">
+  <div class="card mb-3">
+    <!-- fallback mechanism to use default photo is there is no AWS S3 keys or no photos are uploaded from admin -->
+    {% if post.image %}
+    <img class="card-img" src="{{ post.image.url }}" alt="Post image" />
+    {% else %}
+    <img
+      class="card-img"
+      src="/static/blog/images/logo.png"
```

Comment:
shall we adjust the image size ?? feels like it is too big.

Reply:
I will fix it Ama.